tests/README.md

```diff
@@ -33,7 +35,9 @@
     1. Check that all required executables exist.
     2. Execute `tests/run.sh`
 
-    > If want to run one integration test case only, just pass the CASE parameter, such as `make integration_test CASE=sharding`
+    > If want to run one integration test case only, just pass the CASE parameter, such as `make integration_test CASE=sharding`.
+
+    > The online ddl test using pt-osc doesn't work if the upstream MySQL has different connect port and bind port (often caused by port forwarding via NAT). In this case you must specific the real ip and port of MySQL. Othersise you can skip online ddl test by `export ONLINE_DDL_ENABLE=false`.
```

Comment:
```suggestion
    > The online ddl test using pt-osc doesn't work if the upstream MySQL has different connect port and bind port (often caused by port forwarding via NAT). In this case, you must specify the real IP and port of MySQL. Otherwise you can skip online ddl test by `export ONLINE_DDL_ENABLE=false`.
```

and should we write `online ddl` or `online DDL`?